src/sentry/issues/occurrence_consumer.py

```diff
@@ -319,6 +320,21 @@ def process_occurrence_message(
         txn.set_tag("result", "dropped_feature_disabled")
         return None
 
+    # Rate limit based on fingerprint
+    rate_limit_key = f"occurrence_rate_limit:{occurrence_data['fingerprint']}"
```

Comment:
fingerprint is a sequence of strings https://github.com/getsentry/sentry/blob/eacc307d8dfdc4ac95fb5cd6a1d158c79dc35b74/src/sentry/issues/issue_occurrence.py#L30

we'll want to ensure the behavior of this is correct, let's start a slack thread as i'm not quite ure what we want:
1) hash all fingerprints together into one "rate limit fingerprint"
2) rate limit on each fingerprint value separately
3) only rate limit on the first fingerprint value

i'm not sure how frequently issue platform events even have multiple fingerprints, so i'm not sure how major of a consideration this is.

Reply:
checked with dan - the only reason it's a list is because that's what we have for errors, but for occurrences, we only have 1 (all the existing code just gets `fingerprint[0]`) so we should be safe following that and rate limiting on that value.